aztk/spark/client.py

```diff
@@ -20,6 +20,12 @@ def __init__(self, secrets_config):
     Spark client public interface
     '''
     def create_cluster(self, cluster_conf: models.ClusterConfiguration, wait: bool = False):
+        cluster_conf.validate()
+
+        if cluster_conf.user_configuration.username is not None and wait is False:
+            raise error.AztkError(
```

Comment:
I think this is going away soon w/ one of Jake's PRs

Reply:
yeah with the create user on the node right? I just moved the validation here as it is still valid until the other PR get merged

Reply:
I will have to update that PR to account for these changes.